src/rng.rs

```diff
@@ -50,6 +50,24 @@ impl<R: rand::Rng> rand_parsec::Rng for RngParsecCompat<R> {
     }
 }
 
+impl<R: rand::Rng> rand::RngCore for RngParsecCompat<R> {
+    fn next_u32(&mut self) -> u32 {
+        self.0.next_u32()
+    }
+
+    fn next_u64(&mut self) -> u64 {
+        self.0.next_u64()
+    }
+
+    fn fill_bytes(&mut self, dest: &mut [u8]) {
+        self.0.fill_bytes(dest)
+    }
+
+    fn try_fill_bytes(&mut self, dest: &mut [u8]) -> Result<(), rand::Error> {
+        self.0.try_fill_bytes(dest)
+    }
+}
+
```

Comment:
Why do we need this? Isn't this type only passed to parsec and this require only `rand_parsec::Rng` impl?

Reply:
It is also used in mock_parsec :-)

Reply:
Ah, of course :)